.changeset/curvy-doors-promise.md

```diff
@@ -0,0 +1,7 @@
+---
+"docs": patch
+"@fuel-ts/providers": patch
+"@fuel-ts/wallet": patch
```

Comment:
This will be a breaking change, no?

I believe it should be a `minor` bump, then.